.golangci.yaml

```diff
@@ -88,3 +88,8 @@ issues:
       # Workaround to exclude some 'staticcheck' messages, because line-based directive does not seem to work with golangci-lint
       # See https://github.com/golangci/golangci-lint/issues/741#issuecomment-1017014331
       text: 'SA1019: allComponents\[i\].RunningOn is deprecated'
+    - linters:
+        - staticcheck
+      # Workaround to exclude some 'staticcheck' messages, because line-based directive does not seem to work with golangci-lint
```

Comment:
Can you add a TODO note here as a reminder to handle this deprecation once https://github.com/devfile/library/pull/167 is merged?